.github/workflows/tests.yml

```diff
@@ -12,7 +12,7 @@ on:
 env:
   LXD_REQUIRED_TESTS: "storage_buckets"
   LXD_SKIP_TESTS: "clustering_upgrade clustering_upgrade_large"
-  GOCOVERDIR: ${{ ( github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' ) && '/home/runner/work/lxd/lxd/coverage' || '' }}
+  GOCOVERDIR: '/home/runner/work/lxd/lxd/coverage'
```

Comment:
we need this to stay conditional please as we dont need it on the PR runs.

Reply:
it activates the go coverage mode output

Reply:
Whoops, I forgot to revert this back to normal after testing. Did restore the original state just now.